### PR TITLE
11933 ppr total discharges for mhr

### PR DESCRIPTION
**Pull request by thorwolpert**
*Issue #:* /bcgov/entity#11933

*Description of changes:*
Job and framework to run scheduled jobs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: jobs/permanent/ppr-discharges-for-mhr/db/jobs.sql [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+create database jobs;
+create user job_runner WITH PASSWORD 'jw8s0F4';
```
Comment (doug-lovett) [inline jobs/permanent/ppr-discharges-for-mhr/db/jobs.sql]: Password here?

File: jobs/permanent/ppr-discharges-for-mhr/src/ppr_discharges_for_mhr/services/job_tracking/job_tracker.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        except (Exception, psycopg2.DatabaseError) as error:
+            logging.error("Error while closing the job run record", error)
+        finally:
+            if cursor is not None:
+                cursor.close()
+
+
+# Start script
+if __name__ == "__main__":
+    opts = {'host': 'localhost', 'port': 5432, 'dbname': 'jobs', 'user': 'job_runner', 'password': 'jw8s0F4'}
```
Comment (doug-lovett) [inline jobs/permanent/ppr-discharges-for-mhr/src/ppr_discharges_for_mhr/services/job_tracking/job_tracker.py]: Password?

File: jobs/permanent/ppr-discharges-for-mhr/db/jobs.sql (hunk 1 above)
Comment (thorwolpert) [reply jobs/permanent/ppr-discharges-for-mhr/db/jobs.sql]: it's not used anywhere.
i can remove it.

**Review by doug-lovett — APPROVED** [review]
Looks good, just one comment.

**Pull request merged**